ArduCopter/RC_Channel.cpp

```diff
@@ -611,6 +612,12 @@ bool RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const AuxSwi
             }
             break;
 
+        case AUX_FUNC::CUSTOM_CONTROLLER:
+#if AC_CUSTOMCONTROL_MULTI_ENABLED == ENABLED
+            copter.custom_control.set_custom_controller(ch_flag == AuxSwitchPos::HIGH);
+#endif
+            break;
+
```

Comment:
If you define out the whole block users would get the "Invalid channel option" warning. I'm not really sure if that is better or not. I guess better to give a warning than just not working. 

```suggestion
#if AC_CUSTOMCONTROL_MULTI_ENABLED == ENABLED
        case AUX_FUNC::CUSTOM_CONTROLLER:
            copter.custom_control.set_custom_controller(ch_flag == AuxSwitchPos::HIGH);
            break;
#endif
```

Reply:
Although the other channel options don't follow a similar pattern, I agree it is better to give a warning.